applications/nrf5340_audio/unicast_server/README.rst

```diff
@@ -0,0 +1,188 @@
+.. _nrf53_audio_unicast_server_app:
+
+nRF5340 Audio: Unicast server
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF5340 Audio unicast server application implements the :ref:`CIS headset mode <nrf53_audio_app_overview>`.
+
+In this mode, one Connected Isochronous Group (CIG) can be used with two Connected Isochronous Streams (CIS).
+Receiving unidirectional or transceiving bidirectional audio happens using CIG and CIS.
+In addition, Coordinated Set Identification Service (CSIS) is implemented on the server side.
+
+The following limitations apply to this application:
+
+* One CIG, one of the two CIS streams (selectable).
+* Audio output: I2S/Analog headset output.
+* Audio input: PDM microphone over I2S.
+* Configuration: 16 bit, several bit rates ranging from 32 kbps to 124 kbps.
+
+.. _nrf53_audio_unicast_server_app_requirements:
+
+Requirements
+************
+
+The application shares the :ref:`requirements common to all nRF5340 Audio application <nrf53_audio_app_requirements>`.
+
+.. _nrf53_audio_unicast_server_app_ui:
+
+User interface
+**************
+
+This application uses specific mapping for the following user interface elements:
+
+* **VOL-** - Pressed on the headset or the CIS gateway during playback: Turns the playback volume down (and unmutes).
```

Comment:
```suggestion
* **VOL-** - Pressed on the unicast server during playback: Turns the playback volume down (and unmutes).
```
Same for the one below

Reply:
Also add play/pause

Reply:
Do we need this? Isn't PLAY/PAUSE identical to all apps?